plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -134,27 +134,39 @@
 
 {#
   Postgres tables have a maximum length off 63 characters, anything longer is silently truncated.
-  Temp relations add a lot of extra characters to the end of table namers to ensure uniqueness.
+  Temp and backup relations add a lot of extra characters to the end of table names to ensure uniqueness.
   To prevent this going over the character limit, the base_relation name is truncated to ensure
   that name + suffix + uniquestring is < 63 characters.
 #}
-{% macro postgres__make_temp_relation(base_relation, suffix) %}
+
+{% macro postgres__make_relation_with_suffix(base_relation, suffix) %}
```

Comment:
Ok, maybe this macro takes one more argument, `include_dtstring=true`, which controls whether or not the non-deterministic `dtstring` is included in the suffix.

If we care about avoiding a breaking change in the macro signature (for anyone who's overridden this macro locally), it could be:
```sql
{% macro postgres__make_relation_with_suffix(base_relation, suffix) %}
  ...
  {% if kwargs.get('include_dtstring', true %}
    {% set dtstring = dt.strftime("%H%M%S%f") %}
  {% else %}
    {% set dtstring = "" %}
  {% endif %}
{% endmacro %}
```